CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
```

Comment:
Thanks for fixing this!

For a documentation change like this one that isn't user-facing, there's no need to update the changelog. You can add the "no-changelog" label to the PR to bypass the changelog check.

Reply:
Alright, thank you for letting me know.